linear_algebra/tensor_product.lean

```diff
@@ -214,168 +245,156 @@ protected theorem induction_on
   (Cp : ∀ x y, C x → C y → C (x + y)) : C z :=
 quotient.induction_on z $ λ x, free_abelian_group.induction_on x
   C0 (λ ⟨p, q⟩, C1 p q)
-  (λ ⟨p, q⟩ _, show C (-(p ⊗ₜ q)), by rw ← (bilinear M N).neg_left; from C1 (-p) q)
+  (λ ⟨p, q⟩ _, show C (-(p ⊗ₜ q)), by rw ← neg_tmul; from C1 (-p) q)
   (λ _ _, Cp _ _)
 
 section UMP
 
 variables {M N P Q}
-variables (f : M → N → P) (hf : is_bilinear_map f)
-include hf
+variables (f : M →ₗ N →ₗ P)
 
 local attribute [instance] free_abelian_group.lift.is_add_group_hom
-
-def to_module : M ⊗ N → P :=
+/-#check tensor_product.has_scalar
```

Comment:
one more comment. Is this just a left over or a TODO?

Reply:
This is just a left over.